sdk/appconfiguration/azure-appconfiguration/samples/sync_token_samples.py

```diff
@@ -14,11 +14,11 @@
 """
 
 from azure.appconfiguration import AzureAppConfigurationClient
-from util import print_configuration_setting, get_connection_string
+from util import get_connection_string
+from typing import Any, List
 
 
-def handle_event_grid_notifications(event_grid_events):
-    # type: (List[dict[str, Any]]) -> None
+def handle_event_grid_notifications(event_grid_events: List[dict[str, Any]]) -> None:
```

Comment:
Same comment as https://github.com/Azure/azure-sdk-for-python/pull/27894#discussion_r1059551612 -- does this PR make the same changes as #27894, just without opting in to mypy checks?

Reply:
This PR just convert type hints to inline style, will sync with main once that mypy PR merged.